### PR TITLE
Draft: ffwizard project

### PR DESCRIPTION
**Pull request by Akira25**
### Checklist for new or updates project ideas

- [x] I added at least one contact option so interested people can get in touch with me.
- [ ] Data for my project is valid

### CODE REVIEW

**Comment by Akira25** [comment]
@Noki @pktpls @noxilixon I would love, to have you on this too. Either as Mentors or as involved developers. You had big parts in the specification process too.

File: _projects/freifunk-berlin-firmware-wizard.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
 ---
 collaborating_projects:
   - freifunk
+name: "A modular Wizard for OpenWrt"
 desc: "description"
```
Comment (andibraeu) [inline _projects/freifunk-berlin-firmware-wizard.md:5]: please add a short description here

**Comment by noxilixon** [comment]
> @Noki @pktpls @noxilixon I would love, to have you on this too. Either as Mentors or as involved developers. You had big parts in the specification process too.

I would like to join :)  thanks for the initiative.

File: _projects/freifunk-berlin-firmware-wizard.md (hunk 1 above)
Comment (Akira25) [reply _projects/freifunk-berlin-firmware-wizard.md]: Added in newer version of the commit.

File: _projects/freifunk-berlin-firmware-wizard.md [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
 - name: akira25
   contact:
     github: akira25
-    email: "akira@wasweißich.de"
+    email: "martin.hubner@web.de"
     other_chat_options: ["https://matrix.to/#/@akira25:matrix.org"]
-name: "Title for Akira's project idea"
+- name: noki
+  contact:
+    github: noki
```
Comment (Noki) [inline _projects/freifunk-berlin-firmware-wizard.md:27]: `Noki` instead of `noki` please. ;-)

Comment (Akira25) [reply _projects/freifunk-berlin-firmware-wizard.md]: Resolved in current version.

**Pull request merged**